candy.bundle.js

```diff
@@ -2027,19 +2027,22 @@ Candy.Core.Event = function(self, Strophe, $) {
 	 * Notify view that the login window should be displayed
 	 *
 	 * Parameters:
-	 *   (String) presetJid - Preset user JID
+	 *   (String) messageKey - i18n Message Key or null
```

Comment:
Except you're not actually passing the key, you're passing the full translated string.